examples/run_example.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+if command -v python3 &>/dev/null; then
+    echo "Using Python3"
+    python3 server.py &
+
+    python3 client.py &
+    wait $!
+    TEST_EXIT_CODE=$?
```

Comment:
I actually had to disable this test, because my environment had both `python` and `python3` in the path, which is perfectly valid. It picked up `python3` but then tried to use zmq which was only built for python2. I guess it is just on the user to be running in a virtual env that doesn't have both python versions.

Reply:
Good catch, Ok I'll revert to just always use `python...`